### ISSUE TITLE
As an editor, I can upload .tex LaTeX files through the file upload feature.

### ISSUE THREAD

**Issue by mark-bennett-uiowa** [uiowa/uiowa#4844]
On the Physics and Astronomy site, they post LaTeX files for their students to use as templates for research submissions. This will also be the case in Mathematics (and potentially CS in the future). Hoping this could be added to the list of available filetypes for the file upload feature.

Currently we have been wrapping these files inside of ZIP files in order to upload them to the site ([https://physics.uiowa.edu/resources](https://physics.uiowa.edu/resources)):
![Screen Shot 2022-02-21 at 1 57 31 PM](https://user-images.githubusercontent.com/3067017/155019047-f79c8310-8f92-4816-a579-1c1ff2991e85.png)

**Comment by joewhitsitt** [comment]
Just heads up, I read that .tex files can hold nasty things. and that we might want to implement https://github.com/uiowa/uiowa/issues/2264 before allowing more file types. https://www.usenix.org/system/files/login/articles/73506-checkoway.pdf

Also, would this replace #4152 or be in addition to?

**Comment by mark-bennett-uiowa** [comment]
@joewhitsitt I completely understand the concern, and support the AV check implementation prior to this being added.

In regards to #4152, they are two different things. #4152 allows an editor to create something like this inside a text region (see [https://math.uiowa.edu/resources/latex-tex-information](https://math.uiowa.edu/resources/latex-tex-information)):
<img width="821" alt="Screen Shot 2022-02-22 at 11 14 08 AM" src="https://user-images.githubusercontent.com/3067017/155183495-933db242-efcd-40bb-aff4-dfc1b5a6c4c7.png">


where this one just allows the file upload.

**Comment by joewhitsitt** [comment]
Yeah, I was just curious if having an easier way to include .tex as a download would eliminate the need to input the formulas again in a different medium. The one thing I recall from working with LaTeX before is the less you have to replicate/maintain the better.
